src/lib/lite_base/account.ml

```diff
@@ -4,17 +4,39 @@ module Balance = Nat.Make64 ()
 
 module Nonce = Nat.Make32 ()
 
-type t =
-  { public_key: Public_key.Compressed.t
-  ; balance: Balance.t
-  ; nonce: Nonce.t
-  ; receipt_chain_hash: Receipt.Chain_hash.t
-  ; delegate: Public_key.Compressed.t
-  ; participated: bool }
-[@@deriving bin_io, sexp, eq]
+module Stable = struct
+  module V1 = struct
+    module T = struct
+      (* TODO: version *)
+      let version = 1
+
+      let __versioned__ = true
+
+      type t =
+        { public_key: Public_key.Compressed.Stable.V1.t
+        ; balance: Balance.t
+        ; nonce: Nonce.t
+        ; receipt_chain_hash: Receipt.Chain_hash.t
+        ; delegate: Public_key.Compressed.Stable.V1.t
+        ; participated: bool }
+      [@@deriving bin_io, sexp, eq]
```

Comment:
can use `version {asserted}` and remove the definitions of `version` and `__versioned__`.